_data/nav.yml

```
@@ -85,8 +85,10 @@ How to use USWDS:
   - href: /documentation/accessibility/
   - href: /performance/
   - href: /documentation/code-guidelines/
-  - text: Migrating to USWDS 2
+  - text: Migrating to USWDS 3
     href: /documentation/migration/
```

Is migration-v3 the final home of the v3 migration guide, or will it be moved to /migration/
```suggestion
    href: /documentation/migration-V3/
```

---

Apologies - I must have been looking at an old version. Resolved.

---

Related question, should I change the file name from `migration-V3.md` to `migration.md` ?